bridges/HardwareInfoBridge.php

```diff
@@ -0,0 +1,66 @@
+<?php
+class HardwareInfoBridge extends FeedExpander
+{
+	const NAME = 'Hardware Info Bridge';
+	const URI = 'https://HardwareInfo.net/';
+	const DESCRIPTION = 'Tech news from hardware.info (Dutch)';
+	const MAINTAINER = 't0stiman';
+
+	public function collectData()
+	{
+		$this->collectExpandableDatas('https://nl.hardware.info/updates/all.rss', 20);
+	}
+
+	protected function parseItem($feedItem)
+	{
+		$item = parent::parseItem($feedItem);
+
+		//get full article
+		$articlePage = getSimpleHTMLDOMCached('$feedItem->link');
```

Comment:
'$feedItem->link' should be without quotes at all.
That is why bridge errors "Could not resolve host: $feedItem->link"

Reply:
woops... fixed